package.json

```diff
@@ -10,7 +10,8 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "npm run clean && rollup -c",
     "build:watch": "npm run clean && rollup -c -w",
-    "prepublishOnly": "npm run build"
+    "prepublishOnly": "npm run build",
+    "postinstall": "npm run build"
```

Comment:
@gmfun if you want to approve this PR and publish a new `npm` package version I will need to remove this line, as it was only needed when installing this package from github